android/src/main/kotlin/com/hypertrack/sdk/flutter/HyperTrackPlugin.kt

```diff
@@ -60,7 +60,7 @@ public class HyperTrackPlugin(): FlutterPlugin, MethodCallHandler, StreamHandler
     fun registerWith(registrar: Registrar) {
       Log.i(TAG, "registerWith")
 
-      val context = registrar.activity().applicationContext
+      val context = registrar.activity()!!.applicationContext
```

Comment:
@pavel-kuznetsov-hypertrack is it safe to `!!` here (and in other cases bellow asw)? https://kotlinlang.org/docs/null-safety.html#the-operator

Reply:
Let's unwrap it

Reply:
Updating now, will push another commit in few hours